flexmeasures/api/v3_0/sensors.py

```diff
@@ -65,18 +69,37 @@ class SensorAPI(FlaskView):
             "account": AccountIdField(
                 data_key="account_id", load_default=AccountIdField.load_current
             ),
+            "all_accessible": fields.Boolean(required=False, missing=False),
+            "page": fields.Int(
+                required=False, validate=validate.Range(min=1), default=1
+            ),
+            "per_page": fields.Int(
+                required=False, validate=validate.Range(min=1), default=10
+            ),
+            "filter": SearchFilterField(required=False, default=None),
+            "unit": fields.Str(required=False, default=None),
         },
         location="query",
     )
     @permission_required_for_context("read", ctx_arg_name="account")
     @as_json
-    def index(self, account: Account):
+    def index(
+        self,
+        account: Account,
+        all_accessible: bool = False,
+        page: int | None = None,
+        per_page: int | None = None,
+        filter: list[str] | None = None,
+        unit: str | None = None,
```

Comment:
It could be that we have a schema field for valid/supported units. Maybe used in the Sensor Schema? (I'm traveling now...). At least there is some logic w.r.t. to the pint package, which is our unit support. 

I believe a schema field would be great here, so error messages are better.

Reply:
OK so add validation support 👍

I want to clarify though, do you mean field validation (For example: SearchFilterField, AccountIdField etc..) or the actual SensorSchema?

Reply:
A field should suffice. I was just musing where to get inspiration from.

Reply:
Maybe look into flexmeasures.utils.unit_utils

Reply:
Looking into this file I found the `is_valid_unit` function which seems ideal for this scenario. Thank you for the reference! Please feel free to provide any suggestions if my implementation doesn’t fully address the use case.